src/figuresClasses.ts

```diff
@@ -1,11 +1,98 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string = 'triangle';
+
+  color: string;
+
+  constructor(
+    color: string,
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    this.color = color;
+    this.validateSides();
+  }
+
+  private validateSides(): void {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('your error message');
```

Comment:
The error message should be more descriptive. According to the checklist, provide a detailed description of the error, such as 'Side lengths must be greater than zero.'